### ISSUE TITLE
create admin accounts

### ISSUE THREAD

**Issue by hackmastera** [chemheritage/chf-sufia#448]
before admins can do stuff we need to be able to create them.

**Comment by jrochkind** [comment]
It looks like our version of dependencies already have "roles", there's just no UI to create them, or logic to give people with admin 'role' special privs. 

Maybe use this existing architecture? I _think_ it's what hyrax will use. 

~~~
[3] pry(main)> User.first.roles
  Role Load (0.2ms)  SELECT "roles".* FROM "roles" INNER JOIN "roles_users" ON "roles"."id" = "roles_users"."role_id" WHERE "roles_users"."user_id" = ?  [["user_id", 1]]
=> []
[4] pry(main)> Role
=> Role(id: integer, name: string)
~~~

Might be simply:

~~~
admin_role = Role.find_or_create(name: 'admin')
some_admin_user.roles << admin_role
~~~

**Comment by hackmastera** [comment]
@jrochkind yes I added that a while back but didn't finish the implementation.

The wip is documented in the first bullet point here https://github.com/chemheritage/chf-sufia/wiki/List-of-WIP-branches

**Comment by jrochkind** [comment]
@hackmastera I thought that was in an un-merged branch. But 'roles' is in master. So I thought that meant we were getting 'roles' from a dependency; but you added it to master already?

**Comment by jrochkind** [comment]
Ah, maybe that comes from hydra-role-management, which _is_ already a dependency in master?

**Comment by hackmastera** [comment]
@jrochkind It looks like the commits that added hydra-role-management and the admin role definition are still in master. At some point I thought about reverting them, but never merged the branch where I did the reversions.

So those two commits are in master currently; sorry the documentation was unclear.

The admin-role branch is where I picked up again on the work and it was never merged. I think it was ready for some testing, though, and from your earlier comment on one of the commits perhaps a bit of refactoring to pull the logic out of the rake task.
